### ISSUE TITLE
RNN time baby

### ISSUE THREAD

**Issue by rgasper** [rgasper/battery-dev#6]
This looks like a good guide to start from:
https://blog.floydhub.com/a-beginners-guide-on-recurrent-neural-networks-with-pytorch/
Trying out pytorch b/c I've not used it it and I'm a bit tired of mucking around in tensorflow

**Comment by rgasper** [comment]
My first experiment is to throw whole current cycle signals at it all at once - I'm not sure if this is a good idea, but It'll be simpler to train I think. So to do that, first I need to normalize current signal lengths. For funsies, I'm going to do this with an encoder-decoder.
